src/doc/rust.css

```diff
@@ -350,6 +350,14 @@ a.test-arrow {
 	margin-bottom: 1em;
 }
 
+.structfield {  
```

Comment:
Just realized but you didn't modify the correct css file.

Reply:
I have made the exact same changes. I guess I am using a wrong rustdoc installation.

Reply:
If it is wrong, how did you get those screenshots?

Reply:
By modifying the CSS in the web page directly.